invoices/xml/__init__.py

```diff
@@ -144,4 +144,4 @@ def invoice_to_xml(invoice: Invoice) -> etree._Element:
     for tag in tags:
         root.append(tag)
 
-    return root
+    return etree.tostring(root).decode('utf-8')
```

Comment:
I think it's best if `invoice_to_xml` returns an XML Element, this allows us to use xpath and also make the navigation in the hierarchy easier (especially for the tests). If you need a string you can implement another function `xml_to_string` that to converts an XML structure to a string.

Side note: When you change a function, remember to update the typings definition if needed! 😀

Reply:
Okay!